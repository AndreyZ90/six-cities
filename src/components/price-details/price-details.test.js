import React from 'react';
import TestRenderer from 'react-test-renderer';

import PriceDetails from '@/components/price-details/price-details';

const price = 300;

describe(`PriceDetails component snapshot`, () => {
  test(`Should correctly render PriceDetails component`, () => {
    const tree = TestRenderer.create(<PriceDetails price={price} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
