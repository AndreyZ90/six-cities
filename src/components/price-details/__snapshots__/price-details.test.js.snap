// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PriceDetails component snapshot Should correctly render PriceDetails component 1`] = `
<div
  className="property__price"
>
  <b
    className="property__price-value"
  >
    €
    300
  </b>
  <span
    className="property__price-text"
  >
      night
  </span>
</div>
`;
